bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -128,29 +157,52 @@ spec:
           requests:
             cpu: 100m
             memory: 300Mi
-
-      # sbdb: the southbound, or flow DB
+      
+      # sbdb: The southbound, or flow DB. In raft mode 
       - name: sbdb
         image: {{.OvnImage}}
         command:
         - /bin/bash
         - -c
         - |
           set -e
-          if [[ -f "/env/_master" ]]; then
+          if [[ -f /env/_master ]]; then
             set -o allexport
-            source "/env/_master"
+            source /env/_master
             set +o allexport
           fi
-          exec /usr/share/openvswitch/scripts/ovn-ctl run_sb_ovsdb \
+          
+          OVN_NODES_ARRAY=({{.OVN_NODES}})
+
+          if [[ "$K8S_NODE" == "${OVN_NODES_ARRAY[0]}" ]]; then
+            exec /usr/share/openvswitch/scripts/ovn-ctl \
+            --db-sb-cluster-local-port={{.OVN_SB_RAFT_PORT}} \
+            --db-sb-addr=${K8S_NODE} \
+            --db-sb-port={{.OVN_SB_PORT}} \
+            --db-sb-create-insecure-remote=yes \
+            --db-sb-cluster-local-addr=${K8S_NODE} \
             --no-monitor \
             --ovn-sb-db-ssl-key=/ovn-cert/tls.key \
             --ovn-sb-db-ssl-cert=/ovn-cert/tls.crt \
             --ovn-sb-db-ssl-ca-cert=/ovn-ca/ca-bundle.crt \
-            --ovn-sb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off"
-        env:
-        - name: OVN_LOG_LEVEL
-          value: info
+            --ovn-sb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off" \
+            run_sb_ovsdb
+          else
+            exec /usr/share/openvswitch/scripts/ovn-ctl \
+            --db-sb-cluster-local-port={{.OVN_SB_RAFT_PORT}} \
+            --db-sb-cluster-remote-port={{.OVN_SB_RAFT_PORT}} \
+            --db-sb-addr=${K8S_NODE} \
+            --db-sb-port={{.OVN_SB_PORT}} \
+            --db-sb-create-insecure-remote=yes \
```

Comment:
Can we get rid of the `-insecure-remote` options?

Reply:
Indeed we can. Thanks for spotting that